### PR TITLE
initial hunterization

### PR DESCRIPTION
**Pull request by mean-ui-thread**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +24,18 @@ if (POLICY CMP0054)
...
+    URL "https://github.com/cpp-pm/hunter/archive/v0.23.217.tar.gz"
+    SHA1 "4b36c0f8ffc8f3b5718d5e24629fb40c56c99b13"
+)
+
+project(spirv-tools VERSION 2019.4)
+
+hunter_add_package(SPIRV-Headers)
+find_package(SPIRV-Headers CONFIG REQUIRED)
+set(SPIRV-Headers_SOURCE_DIR ${SPIRV-HEADERS_ROOT})
+
```
Comment (rbsheth) [inline CMakeLists.txt:38]: Can you comment out the following lines?
```
set(CMAKE_POSITION_INDEPENDENT_CODE ON)
set(CMAKE_CXX_STANDARD 11)
```
They should be set by the toolchain.

Comment (mean-ui-thread) [reply CMakeLists.txt:38]: you're correct. I missed them. Thanks.

Comment (mean-ui-thread) [reply CMakeLists.txt:38]: done.

**Review by rbsheth — APPROVED** [review]

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/SPIRV-Tools/releases/tag/v2019.4-p0

**Pull request merged**